backend/proxy_server/auth.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/github"
+)
+
+// global oauth providers' client config
+var (
+	githubConfig *oauth2.Config
+)
+
+// providers' client parameters
+const (
+	GithubClientID          = "8a2c4750b721d684d680"
+	GithubClientSecret      = "ad444449bb7f9478c7c1f88bebd69b3c19274b72"
+	GithubClientRedirectURL = "http://localhost/login"
```

Comment:
Please replace this with the production server IP. And we should mention that "this should be changed accordingly" in the `get started` section of README.